docs/AWS_Integration_Guide/Configuring_AWS_integration.md

```diff
@@ -51,35 +51,16 @@ Then follow the steps below:
-            - Go to **AWS Identity and Access Management**, click **Users** and select your user name from the list.
-            - Select the **Security credentials** tab and, under the **Access keys** section, click **Create access key**.
-               ![Create access key button](images/create_access_key.png "Create access key button")
-            - Copy the generated **Access key ID** and **Secret access key**.
-               ![Copy access key and ID](images/aws_access_key.png "Copy access key and ID")
-            - In Coiote DM, go to the **Profiles** tab of your integration group and paste the credentials as values for `AWSaccessKeyID` and `AWSsecretAccessKey`.
-            - Click **Save**.
-        - For `AWSregion`:
-            - While in AWS IoT Core, click on your region name in the top navigation bar to expand the list of regions. Then, copy the hyphenated region name (e.g. *us-east-1*).
-              ![Copy region name](images/region_name.png "Copy region name")
-            - In Coiote DM, go to the **Profiles** tab of your integration group and paste the region name as the value for `AWSregion`.
-            - Click **Save**.
-        - For `AWScontrolPlaneEndpointAddress`:
-            - Go to **AWS documentation**: <https://docs.aws.amazon.com/general/latest/gr/iot-core.html>.
-            - From the **Control Plane API Endpoints** section, find the endpoint that matches your region (e.g. *iot.us-east-1.amazonaws.com*) and copy it.
-            - In Coiote DM, go to the **Profiles** tab of your integration group and paste the credentials as the value for `AWScontrolPlaneEndpointAddress`.
-            - Click **Save**.
-        - For `AWSdataPlaneEndpointAddress`:
-            - Open your command line and run the following command:
-              ```
-              aws iot describe-endpoint --endpoint-type iot:Data-ATS
-              ```
-            - Copy the returned result.
-              ![Copy data plane endpoint address](images/dataplane.png "Copy data plane endpoint address")
-            - In Coiote DM, go to the **Profiles** tab of your integration group and paste the result as the value for `AWSdataPlaneEndpointAddress`.
-            - Append `:8443` port to the pasted value.
-            - Click **Save**.
+- Go to **Device groups**, select your custom integration group (or the **AWSiotCoreCertAuth** group, depending on the previous step) and go to **Profiles**. Complete the **AWS** setting value:
+  - Open your command line and run the following command:
+           ```
+           aws iot describe-endpoint --endpoint-type iot:Data-ATS
+           ```
+  - Copy the returned result.
+  ![Copy data plane endpoint address](images/dataplane.png "Copy data plane endpoint address")
+  - In Coiote DM, go to the **Profiles** tab of your integration group and paste the result as the value for `AWSdataPlaneEndpointAddress`.
+  - Append `:8443` port to the pasted value.
```

Comment:
Can't we do this automatically in the tasks?

Reply:
we could, but we've decided not to write the same port in every task